docs/modules/ROOT/pages/references/zones.adoc

```diff
@@ -15,3 +15,19 @@ Integrated Cluster Registry Domain:: registry.cloudscale-lpg-2.appuio.cloud
 SDN:: https://cilium.io/[Cilium]
 Outgoing IPv4:: 185.98.123.195
 CNAME:: cname.cloudscale-lpg-2.appuio.cloud.
+
+== exoscale.com - CH-GVA-2 0
+
+____
+Direct link to the https://console.exoscale-ch-gva-2-0.appuio.cloud/[OpenShift Console^]
+____
+
+Cloud Provider:: https://www.exoscale.com/[exoscale.com] / https://www.exoscale.com/datacenters/switzerland/#ch-gva-2[Zone CH-GVA-2] / https://map.geo.admin.ch/?zoom=10&E=2496579&N=1118315&crosshair=marker[Geneva]
+OpenShift Version:: 4.9
+Kubernetes Version:: 1.22
+Kubernetes API endpoint:: https://api.exoscale-ch-gva-2-0.appuio.cloud:6443
+Default App Domain:: apps.exoscale-ch-gva-2-0.appuio.cloud
+Integrated Cluster Registry Domain:: registry.exoscale-ch-gva-2-0.appuio.cloud
+SDN:: https://cilium.io/[Cilium]
+Outgoing IPv4:: 89.145.167.35
```

Comment:
We actually don't have a single egress IP on Exoscale, but rather traffic from each node uses that node's public IP as the egress IP, cf.

```bash
simon@phoenix:~ [appuio-ch-gva-2-0] $ kubectl exec -it shell-54bf44bdc5-bxcnx -- curl ip.simutech.io
194.182.161.247
simon@phoenix:~ [appuio-ch-gva-2-0] $ kubectl exec -it shell-54bf44bdc5-n4pgf -- curl ip.simutech.io
85.217.160.230
simon@phoenix:~ [appuio-ch-gva-2-0] $ kubectl exec -it shell-54bf44bdc5-pvwlr -- curl ip.simutech.io
85.217.160.220
simon@phoenix:~ [appuio-ch-gva-2-0] $ kubectl get pods -o wide
NAME                     READY   STATUS    RESTARTS   AGE     IP            NODE          NOMINATED NODE   READINESS GATES
shell-54bf44bdc5-bxcnx   1/1     Running   0          2m50s   10.0.11.154   worker-0f6e   <none>           <none>
shell-54bf44bdc5-n4pgf   1/1     Running   0          3m2s    10.0.10.153   worker-175e   <none>           <none>
shell-54bf44bdc5-pvwlr   1/1     Running   0          2m50s   10.0.9.228    worker-5e68   <none>           <none>
```